docs/components/console/manage-organization/switch-organization.md

```diff
@@ -5,4 +5,6 @@ title: Switch organization
 
 If a user is assigned to more than one organization, the organization can be changed in the menu of the navigation bar.
 
+<!--- By clicking Manage?--->
```

Comment:
+1, this needs more clarification and a high-quality text description of the screenshot.

Reply:
Resolved in latest commit.